snuba/consumers/consumer_builder.py

```diff
@@ -39,7 +40,7 @@ def __init__(
         replacements_topic: Optional[str],
         max_batch_size: int,
         max_batch_time_ms: int,
-        bootstrap_servers: Sequence[str],
+        bootstrap_servers: Optional[str],
```

Comment:
You are still passing a sequence here in the consumer.py file so this does not type check now. I don't think you need this change at all.

Reply:
it may be a left over from previous refactoring. let me double check